sentences/en/media_player_HassMediaPrevious.yaml

```diff
@@ -0,0 +1,10 @@
+language: en
+intents:
+  HassMediaPrevious:
+    data:
+      - sentences:
```

Comment:
Maybe also use "play"/"replay" as verbs.

Reply:
And I think just `previous track on <name>` should work, without the verb

Reply:
Have a look at what I have changed here and see if I have captured what you were thinking.

Reply:
Not sure about omitting the verb, that wouldn't be correct grammat and I thought that was desired....

Reply:
@synesthesiam any thoughts on this?

Reply:
Just this question remains